MaxText/configs/base.yml

```diff
@@ -121,13 +122,14 @@ num_slice: 1 # used by run-sweep.py . will be copied to the other yml file.
 dataset_path: "gs://max-datasets-rogue-useast/"
 
 vocab_size: 32_768 # powers of 2 for sharding
-vocab_relative_path: "vocabs" # Assumes we're allowed
-dataset_name: "c4/en:3.0.1"
-eval_dataset_name: "c4/en:3.0.1"
-eval_split: "validation"
-per_device_batch_size: 4
+vocab_relative_path: "vocabs"  # Assumes we're allowed
+dataset_name: 'c4/en:3.0.1'
+eval_dataset_name: 'c4/en:3.0.1'
+eval_split: 'validation'
+per_device_batch_size: 12.0
```

Comment:
Let's set it to -1 to force setting explicitely

Reply:
Done.